src/models/message/LibraryResponse.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Estonian Information System Authority
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import { SignatureAlgorithm } from "../SignatureAlgorithm";
+
+export interface LibraryStatusResponse {
```

Comment:
Shouldn't these have a common base class with `ExtensionResponse` to avoid code and comment duplication? I'm not sure this is a good idea, shooting it out nevertheless.

Reply:
The next PR actually changes things, so refactored it there https://github.com/web-eid/web-eid.js/pull/17/files#diff-b5162f22e530e525184793ea4dfa46c3af669d74266822913771b9d4680d8101R30-R33